tests/Query/Filter/TagsField.test.ts

```diff
@@ -673,4 +673,12 @@ describe('grouping by tag', () => {
             "
         `);
     });
+
+    it('should sort groups for TaskField', () => {
+        const grouper = new TagsField().createNormalGrouper();
+        const taskLines = ['- [ ] a #tag1', '- [ ] a #tag2', '- [ ] a #tag1 #tag2', '- [ ] a'];
+        const tasks = taskLines.map((taskLine) => fromLine({ line: taskLine }));
+
+        expect({ grouper, tasks }).groupHeadingsToBe(['(No tags)', '#tag1', '#tag2']);
+    });
```

Comment:
If we stick with the idea of having a helper function that creates a selection of tasks with a wide range of values, I think it would be good for this to also include some nested tags to see how they sort...

Reply:
Ah right! I had a feeling that I need to add more to this test! Thanks =)

Done.